### PR TITLE
Refactor/harvest gateway test

### PR DESCRIPTION
**Pull request by C-gyorfi**

### CODE REVIEW

File: CryptoTechReminderSystem.Test/HarvestGatewayTests.cs [hunk 1]
```diff
@@ -13,117 +12,136 @@ public class HarvestGatewayTests
...
-        [Test]
-        public void CanGetDeveloper()
-        {
-            SetUpUsersApiEndpoint(
-                "123",
-                "Wen Ting",
-                "Wang",
-                "ting@email.com"
-            );
+            private FluentSimulator _fluentSimulator;
```
Comment (chubberlisk) [inline CryptoTechReminderSystem.Test/HarvestGatewayTests.cs]: I think we should rename this to `harvestApi` like how we've done in the acceptance test.

File: CryptoTechReminderSystem.Test/HarvestGatewayTests.cs [hunk 2]
```diff
@@ -13,117 +12,136 @@ public class HarvestGatewayTests
...
-                "Tingker",
-                "Bell",
-                "tingkerbell@email.com"
-            );
-            
-            var response = _harvestGateway.RetrieveDevelopers();
+            [SetUp]
+            public void Setup()
+            {
+                _fluentSimulator = new FluentSimulator(Address);
```
Comment (Rumaysa) [inline CryptoTechReminderSystem.Test/HarvestGatewayTests.cs]: `_fluentSimulator` can be renamed to `_harvestAPI` as in previous tests

File: CryptoTechReminderSystem.Test/HarvestGatewayTests.cs [hunk 3]
```diff
@@ -13,117 +12,136 @@ public class HarvestGatewayTests
...
-        public void CanGetATimeSheet()
-        {
-            SetUpUsersTimeSheetApiEndpoint("../../../HarvestTimeEntriesApiEndpoint.json");
-            
-            var response = _harvestGateway.RetrieveTimeSheets();
-            
-            response.First().UserId.Should().Be(1782975);
+            private void SetUpUsersApiEndpoint(string id, string firstName, string lastName, string email)
+            {
+                var json = $"{{  \"users\":[    {{      \"id\":{id},      \"first_name\":\"{firstName}\"" +
```
Comment (chubberlisk) [inline CryptoTechReminderSystem.Test/HarvestGatewayTests.cs]: Should we use a JSON file like we've done for the Slack Gateway tests?

File: CryptoTechReminderSystem.Test/HarvestGatewayTests.cs (hunk 1 above)
Comment (C-gyorfi) [reply CryptoTechReminderSystem.Test/HarvestGatewayTests.cs]: Resolved in https://github.com/chubberlisk/crypto_tech/pull/14/commits/75cfa6f2ab7519f8774e13d0dbef8f96f80b0a73

File: CryptoTechReminderSystem.Test/HarvestGatewayTests.cs (hunk 2 above)
Comment (C-gyorfi) [reply CryptoTechReminderSystem.Test/HarvestGatewayTests.cs]: Resolved in https://github.com/chubberlisk/crypto_tech/pull/14/commits/75cfa6f2ab7519f8774e13d0dbef8f96f80b0a73

**Pull request merged**